hypoddpy/hypodd_relocator.py

```diff
@@ -384,9 +384,9 @@ def _write_catalog_input_file(self, phase_weighting=None):
                         station_id=pick["station_id"])
                     self.log(msg, level="warning")
                     continue
-                weight = phase_weighting(pick['station_id'], pick['phase'],
+                weight = phase_weighting((pick['station_id'], pick['phase'],
                                          pick['pick_time'],
-                                         pick['pick_time_error'])
+                                         pick['pick_time_error']))
```

Comment:
Oh wow - blind me - yea the lambda definition is wrong from me...On Python 3 the current syntax is illegal...on Python 2 it actually unpacks the tuple to its arguments. Kind of nice that this works but it is IMHO slightly confusing.

While you are at it: Can you please change the lambda function to:

```python
self.phase_weighting = lambda sta_id, ph_type, time, uncertainty: 1.0
```